### ISSUE TITLE
Error: Cannot find module './threshold-checker'

### ISSUE THREAD

**Issue by phuson** [cb1kenobi/gulp-babel-istanbul#7]
It seems that the `threshold-checker.js` file is not being included in the npm package. Here's the error I'm getting:

```
Error: Cannot find module './threshold-checker'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/repo/node_modules/gulp-babel-istanbul/index.js:11:15)
    at Module._compile (module.js:541:32)
    at Module._extensions..js (module.js:550:10)
    at Object.require.extensions.(anonymous function) [as .js] (/repo/node_modules/babel-register/lib/node.js:166:7)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
```

Possibly because only index.js is included in the `files` property in package.json? https://github.com/cb1kenobi/gulp-babel-istanbul/blob/master/package.json#L28

**Comment by alansouzati** [comment]
My CI is currently blocked on that: https://travis-ci.org/grommet/grommet-blog/builds/132345460

**Comment by alansouzati** [comment]
It seems to be related to the new file added: threshold-checker.

For some reason it is missing in the `1.3.0`, initially I thought that there was a rule inside `.npmignore` affecting this file, but I could not find any. Maybe whoever triggered the `npm publish` missed this file somehow?

**Comment by alansouzati** [comment]
Actually it makes sense @phuson, I believe the file key needs to be removed, as you guys are using .npmignore already

**Comment by cb1kenobi** [comment]
Sorry guys. Fixed now. The original `gulp-istanbul` project this forks used the `files` section of the `package.json` to whitelist which files should be packages instead of blacklisting using the `.npmignore`. I removed the `files` section and published v1.3.1.
